ledger.go

```diff
@@ -321,28 +318,6 @@ func (l *Ledger) Find(query string, max int) (results []string) {
 	return append(results, l.indexer.Find(query, count)...)
 }
 
-// PushSendQuota permits one token into this nodes send quota bucket every millisecond
-// such that the node may add one single transaction into its graph.
-func (l *Ledger) PushSendQuota() {
```

Comment:
aren't we planning to use it later?

Reply:
It would be best for us to keep the rate limiting outside of `ledger` and inside the APIs instead.